web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -265,7 +265,7 @@
                 <%--table--%>
```

Comment:
I do not know when this comment created, but this comment does not share too much information. Could you please delete it?